main.nf

```diff
@@ -62,6 +62,14 @@ params.vep_genome            = WorkflowMain.getGenomeAttribute(params, 'vep_geno
 params.vep_species           = WorkflowMain.getGenomeAttribute(params, 'vep_species')
 params.vep_version           = WorkflowMain.getGenomeAttribute(params, 'vep_version')
 
+/*
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    ALTERNATIVE INPUT FILE ON RESTART
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+*/
+
+params.input_restart = WorkflowSarek.retrieveInput(params, log)
```

Comment:
Here we define the new parameter `input_restart` which will be used if input is an empty string